_posts/2019-03-05-marfeel-event.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: 4 Ideas from the Barcelona Prebid Summit
+description: 
+permalink: /blog/marfeel-event-2019
+---
+
+
+<br>
+
+(Originally [posted on Marfeel's site](https://www.marfeel.com/blog/prebid-leadership-summit-2019/))
+
+![Barcelona Summit](/assets/images/blog/2019-03-05-marfeel-event-1.jpg){: .pb-lg-img :}
+
+As Mobile World Congress has brought the digital publishing world to us this week, we welcomed our industry partner Prebid.org to the Marfeel’s offices to host their third ever Prebid Leadership Summit. Led by the Chair of Prebid, Alexandra Smith, the mission of the day was to understand the changes leading the header bidding industry, and the role of the Prebid organization in driving these changes.
+
+We were also joined by Remi Boudard from AppNexus, Enrique Blanc from Grupo Zeta, and Lucas Isern and Xavi Beumala from Marfeel, discussing the rise and challenges of adopting server-side header bidding.
+
+To recap the key points for those that weren’t able to attend, we’ve collated everything publishers need to know.
+
+Here are the key takeaways:
+
+## 1. Header Bidding is still in the early-adopter phase, especially for Server Side
+
+In Europe in 2017, 10-15% of display impressions were generated using header bidding. By the end of 2018, it was closer to 50%. Client-side is still the dominant method of implementation.
+
+Client-side header bidding was always supposed to be a temporary phenomenon before a rapid migration to server-side. This was due to the impact of page-load on the user experience and the efficiency gains provided by server-side.
+
+But, there are still limitations that prevent server-side from being rolled out across advertisers. One of the main challenges for adoption is user identification. When you add an additional step for cookie matching, match rates can go down and in turn, impact monetization. Buyers only bid high when they know who the user is, so losing match rates can severely impact monetization for many. Until universal ID solutions come to fruition, the industry is hindered.
+
+Whatsmore, web publishers already have working systems for display. There is not currently enough incentive to apply wholesale changes to their approach. They don’t want to spend time and money shifting to a system that will be more advanced and more rigorously industry-tested in a year or so.
```

Comment:
I understand if we want to just reproduce the blog post exactly as it appears, but whatsmore is not one word.